web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -298,16 +367,21 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
       setLoading(true);
       const res = await getReviewPeriods(csrf);
       const data =
-        res &&
-        res.payload &&
-        res.payload.data &&
-        res.payload.status === 200 &&
-        !res.error
+        res?.payload?.data && res.payload.status === 200 && !res.error
           ? res.payload.data
           : null;
       if (data) {
         dispatch({ type: UPDATE_REVIEW_PERIODS, payload: data });
         setLoading(false);
+      } else {
+        console.log(res?.error);
```

Comment:
Nice! If you also want to keep the console output, I would change it to use `console.error` instead of `console.log`. That will make it appear in red.